.github/workflows/pr-checks.yml

```diff
@@ -15,8 +15,13 @@ on:
       - demilestoned
 concurrency:
   group: pr-checks-${{ github.event.number }}
+permissions:
+  contents: write
 jobs:
   main:
+    permissions:
+      actions: write
+      contents: read
```

Comment:
I think this check only requires:

```yaml
permissions:
  contents: read
  pull-requests: write
```

@marefr or @zerok might know more?

Reply:
My guess is this was the code that mattered:
https://github.com/grafana/grafana-github-actions/blob/6834fcbfb10a065f2fd6e20f04417252a9ca2397/pr-checks/Dispatcher.ts#L78-L83
https://docs.github.com/en/rest/commits/statuses?apiVersion=2022-11-28#create-a-status

Reply:
Dropped in https://github.com/grafana/grafana/pull/72503#event-9948642305: https://github.com/grafana/grafana/compare/89f76cea1c02015c1e6cbd9164e447584df2c2cd..5befafda553b4b7c4e46ed2af55fc772254f1dc9 (GitHub doesn't seem to be properly updating the PR state even though the push happened -- I trip on this sort of thing occasionally -- GitHub aims for "eventual consistency"...)

Reply:
👍🏻  It looks like `statuses: write` will be the correct permission for that then? 

https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs#overview

For clarity for others, by default it defaults to read/write https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token

Reply:
If it's status that was the write, yes. I'm not really sure what I saw at the time.

Fwiw, the default permissions can be different by org/enterprise (my org where I do work defaults to `read` in general which is why this PR / issue exists): 
https://docs.github.com/en/enterprise-cloud@latest/organizations/managing-organization-settings/disabling-or-limiting-github-actions-for-your-organization#configuring-the-default-github_token-permissions

Reply:
Yup - I meant that more as a "we're not escalating _our_ default permissions, just making the defaults explicit" 👍🏻

Reply:
For the record, it appears that `statuses` was indeed necessary, on the bright side, we did talk about it. https://github.com/grafana/grafana/commit/3172715a025639e09b4474a0a3a855cadcd23f64